### DISCUSSION TITLE
open-shell custom start menu item for Snipping Tool

### DISCUSSION THREAD

**Discussion by noahpeal** [Open-Shell/Open-Shell-Menu#1943] [Q&A]
in windows 11, i'd like to add an open-shell custom start menu item that runs Snipping Tool, which is in a protected directory. the full path for this is:

C:\Program Files\WindowsApps\Microsoft.ScreenSketch_11.2405.32.0_x64__8wekyb3d8bbwe\SnippingTool

... but because WindowsApps is protected it can't be run from a command line. however i can create a desktop shortcut that runs it.

how can i run this from my open-shell start menu?

see also this:
https://answers.microsoft.com/en-us/windows/forum/all/location-of-windows-snipping-tool-exe-file/cb112041-71b8-4f4c-83e9-6f54618c9958

**Comment by ge0rdi** [comment]
You can _search_ for `Snipping Tool` and then pin it to Start menu using `Pin to Start menu (Open-Shell)` from context menu.

**Reply by noahpeal** [reply]
tx..... and sorry but you lost me. a "search for Snipping Tool" will fail because it's in a secure directory that can't be searched. no idea what " pin it to Start menu using Pin to Start menu (Open-Shell) from context menu" means. a bit more detail would be helpful.

**Reply by ge0rdi** [reply]
> a "search for Snipping Tool" will fail because it's in a secure directory that can't be searched

In `Open-Shell` _search box_ write `snip`. You should see `Snipping Tool` in search results:
![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/2453094/1e6c7074-7acf-4717-a761-7c97dc233917)

> no idea what " pin it to Start menu using Pin to Start menu (Open-Shell) from context menu" means.

Right-click on the `Snipping Tool` item and select `Pin to Start menu (Open-Shell)`.

**Comment by noahpeal** [comment]
tx again but.... nope. here's what i get, below (sorry about enormous size). i wouldn't expect this to return any results since, eg, dir from a command prompt can't find Snipping Tool because it's in WindowsApps which is a protected directory. and i've read that changing permissions on that directory can break things.

![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/174548362/9c99214c-bd4a-4b00-ac0c-64bb20d83ca0)

**Reply by ge0rdi** [reply]
What I meant is to use search in Start menu:
![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/2453094/ce9b29ec-5b14-48c0-9c5f-aa0aa1d32741)
